PayPal/src/test/java/com/braintreepayments/api/paypal/PayPalVaultRequestUnitTest.java

```diff
@@ -1,26 +1,26 @@
 package com.braintreepayments.api.paypal;
 
+import static junit.framework.Assert.assertEquals;
+import static junit.framework.Assert.assertFalse;
+import static junit.framework.Assert.assertNull;
+import static junit.framework.Assert.assertTrue;
+import static org.junit.Assert.assertSame;
+import static org.mockito.Mockito.mock;
+
 import android.os.Parcel;
 
 import com.braintreepayments.api.core.Authorization;
 import com.braintreepayments.api.core.Configuration;
 import com.braintreepayments.api.core.PostalAddress;
-import com.braintreepayments.api.paypal.PayPalLineItem;
-import com.braintreepayments.api.paypal.PayPalRequest;
-import com.braintreepayments.api.paypal.PayPalVaultRequest;
 
 import org.json.JSONException;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.robolectric.RobolectricTestRunner;
 
 import java.util.ArrayList;
-
-import static junit.framework.Assert.assertEquals;
-import static junit.framework.Assert.assertFalse;
-import static junit.framework.Assert.assertNull;
-import static junit.framework.Assert.assertTrue;
-import static org.mockito.Mockito.mock;
+import java.util.List;
+import java.util.Objects;
 
 @RunWith(RobolectricTestRunner.class)
 public class PayPalVaultRequestUnitTest {
```

Comment:
We don't currently have robust testing of our JSON parameter formatting - we rely on demo app to test that the request is properly formatted. Is there a good way to test the JSON body of these requests or should we wait until GW sandbox verification via the demo app (future ticket)?

Reply:
I think we could write a test around calling `toJson()` on a given `PayPalVaultRequest` and verifying the JSON string that's returned. Is there additional testing that we'd want to do?